client/verta/verta/_deployment/endpoint.py

```diff
@@ -9,26 +9,24 @@
 
 
 class Endpoint(object):
-    def __init__(self, conn, conf, workspace, id):
-        self._conn = conn
-        self._conf = conf
-
+    def __init__(self, conn, conf, workspace, id, path):
         self.workspace = workspace
         self.id = id
+        self._path = path
```

Comment:
I would strongly prefer to not change this (suggesting revert), to keep a consistent internal interface compared to our other objects (`Project`, `RegisteredModel`)
```suggestion
    def __init__(self, conn, conf, workspace, id):
        self._conn = conn
        self._conf = conf

        self.workspace = workspace
        self.id = id
```

Reply:
removed path